finance-portal/values.yaml

```diff
@@ -0,0 +1,83 @@
+service:
+    type: ClusterIP
+
+imageCredentials:
+    enabled: false
+    name: pull-secrets
+
+db:
+    protocol: 'mysql'
+    host: '$release_name-centralledger-mysql'
+    port: '3306'
+    user: 'central_ledger'
+    password: 'oyMxgZChuu'
+    schema: 'central_ledger'
+
+config:
+    settlementsEndpoint: 'centralsettlement/v1'
+    centralLedgerEndpoint: 'centralledger-service:3001'
+    settlementManagementEndpoint: 'settlement'
+    settlementManagementPort: '5000'
+    isOauthEnabled: false
+    oauthServer: 'https://localhost'
+    oauthPort: 9443
+    oauthLoginPath: '/oauth2/token'
+    oauthTokenValidationPath: '/oauth2/introspect'
+    oauthUserInfo: '/oauth2/userinfo'
+    oauthTokenRevoke: '/oauth2/revoke'
+    oauthClientKey: 'testkey'
+    oauthClientSecret: 'testsecret'
+    BypassAuth: 'true'
+    InsecureCookie: 'false'
+    is_populate:
+      name: ispopulate
+      http:
+        endpoint: 'something'
+        timeout: 30000
+      authentication:
+        type: 'basic'
+        credentials:
+          username: 'admin'
+          password: 'admin'
+    jasperUser: 'user'
+    jasperPassword: 'password'
+    report1: "something"
+    report2: "something"
+
+frontend:
+  image:
+    repository: mojaloop/finance-portal-ui
+    tag: v0.1.0
+    pullPolicy: Always
+    port: 80
+  ingress:
+    enabled: true
+    externalPath: /
+    hosts:
+      api: finance-portal.local
+    annotations:
+      nginx.ingress.kubernetes.io/rewrite-target: '/'
+    tls:
+
+backend:
+  image:
+    repository: mojaloop/finance-portal-backend-service
+    tag: v0.0.1
+    pullPolicy: Always
+    port: 3000
+  init:
+    enabled: false
+    image:
+      repository: mojaloop/finance-portal-backend
```

Comment:
I think this should be `mojaloop/finance-portal-backend-service`, as the docker pull for the license scanner can't find it

Reply:
You are correct @lewisdaly, I updated it.